projects/dashboards/data-atlas/src/app/app.component.ts

```diff
@@ -1,15 +1,15 @@
 import {ChangeDetectionStrategy, Component, ViewContainerRef} from '@angular/core';
 import {MatIconRegistry} from '@angular/material/icon';
 import {DomSanitizer} from '@angular/platform-browser';
-import {Router} from '@angular/router';
+import {Router, RouterOutlet} from '@angular/router';
 import {UserService} from '@geoengine/common';
 
 @Component({
     selector: 'geoengine-root',
     templateUrl: './app.component.html',
     styleUrls: ['./app.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    standalone: false,
+    imports: [RouterOutlet],
```

Comment:
unrelated to this line:

- what about occurrences of: standalone: true (ctrl+f), can we remove them?
- what about remaining occurences of: standalone: false (ctrl+f) in the code, are they still correct?

Reply:
- Removed standalone: true
  - [d0aacc9](https://github.com/geo-engine/geoengine-ui/pull/795/commits/d0aacc9174f1afdb4085e9ccee34dce235784dce)
- There is no standalone: false